packages/e2e-tests/specs/experiments/multi-entity-editing.test.js

```diff
@@ -41,9 +41,12 @@ const createTemplatePart = async (
 	);
 	await openDocumentSettingsSidebar();
 
-	const nameInputSelector =
-		'.block-editor-block-inspector .components-text-control__input';
-	const nameInput = await page.waitForSelector( nameInputSelector );
+	const advancedPanelXPath = `//div[contains(@class,"interface-interface-skeleton__sidebar")]//button[@class="components-button components-panel__body-toggle"][contains(text(),"Advanced")]`;
+	const advancedPanel = await page.waitForXPath( advancedPanelXPath );
+	await advancedPanel.click();
```

Comment:
I wonder if current e2e utils `ensureSidebarOpened`, `findSidebarPanelToggleButtonWithTitle`, `findSidebarPanelWithTitle`, may be useful here?

Reply:
Would be great, but just checked them and they are only working in `edit-post` context 😞

Reply:
> are only working in edit-post context

Isn't that the context this function is being run in?  It looks like it is used in test setup in before all, after `createNewPost` and before `siteEditor.visit()`.  Similarly it uses `openDocumentSettingsSidebar` a few lines above this which is also reliant on `edit-post*` selectors.